libp2p/protocols/pubsub/rpc/messages.nim

```diff
@@ -27,52 +27,52 @@ proc expectedFields[T](t: typedesc[T], existingFieldNames: seq[string]) {.raises
     raise newException(CatchableError, $T & " fields changed, please search for and revise all relevant procs. New fields: " & $fieldNames)
 
 type
-    PeerInfoMsg* = object
-      peerId*: PeerId
-      signedPeerRecord*: seq[byte]
-
-    SubOpts* = object
-      subscribe*: bool
-      topic*: string
-
-    MessageId* = seq[byte]
-
-    Message* = object
-      fromPeer*: PeerId
-      data*: seq[byte]
-      seqno*: seq[byte]
-      topicIds*: seq[string]
-      signature*: seq[byte]
-      key*: seq[byte]
-
-    ControlMessage* = object
-      ihave*: seq[ControlIHave]
-      iwant*: seq[ControlIWant]
-      graft*: seq[ControlGraft]
-      prune*: seq[ControlPrune]
-      idontwant*: seq[ControlIWant]
-
-    ControlIHave* = object
-      topicId*: string
-      messageIds*: seq[MessageId]
-
-    ControlIWant* = object
-      messageIds*: seq[MessageId]
-
-    ControlGraft* = object
-      topicId*: string
-
-    ControlPrune* = object
-      topicId*: string
-      peers*: seq[PeerInfoMsg]
-      backoff*: uint64
-
-    RPCMsg* = object
-      subscriptions*: seq[SubOpts]
-      messages*: seq[Message]
-      control*: Option[ControlMessage]
-      ping*: seq[byte]
-      pong*: seq[byte]
+  PeerInfoMsg* = object
+    peerId*: PeerId
+    signedPeerRecord*: seq[byte]
+
+  SubOpts* = object
+    subscribe*: bool
+    topic*: string
+
+  MessageId* = seq[byte]
+
+  Message* = object
+    fromPeer*: PeerId
+    data*: seq[byte]
+    seqno*: seq[byte]
+    topic*: string
+    signature*: seq[byte]
+    key*: seq[byte]
+
+  ControlMessage* = object
+    ihave*: seq[ControlIHave]
+    iwant*: seq[ControlIWant]
+    graft*: seq[ControlGraft]
+    prune*: seq[ControlPrune]
+    idontwant*: seq[ControlIWant]
+
+  ControlIHave* = object
+    topic*: string
```

Comment:
The gossipsub fields are named different from floodsub (https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.0.md#protobuf)

Reply:
Oh, alright. That I did not check, thanks for the heads up.